docs/pages/careers/code-infra-engineer.md

```diff
@@ -0,0 +1,139 @@
+- Migrate the docs platform to the Next.js App Router
+- Allow users to search the docs within a specific product scope
+- Make it possible to store different parts of our documentation in different repositories
+- Introduce a more scalable solution for hosting videos and images
+
+### The best parts of this job
+
+- **You'll be at the cutting edge of application development** — supporting one of the fastest-growing UI libraries on the market.
+- **You'll be part of an active, open, friendly community** of developers that are excited about building awesome applications.
+
+### The worst parts of this job
+
+- **Shifting context.**
+  You will necessarily have to shift context and dive into a different feature before the current one is done.
+  It may even be in an area of the codebase you're unfamiliar with or don't have a ton of understanding about.
+  It's fun, rewarding work, but it can be very challenging.
+- **We move quickly but don't sacrifice quality**.
+  We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **MUI has a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## Who we're looking for
+
+We're looking for someone with strong technical skills and experience in infrastructure related work. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
+
+### Required
+
+- **Expertise in the modern JavaScript ecosystem**.
+  MUI's open-source projects are built on the shoulders of giants, making use of technologies such as ES2021, TypeScript, Node.js, React, Next.js, webpack, and Babel.
+- **A track record of demonstrating an eye for product and solving real-world user problems**. If you have a knack for solving problems at the root cause, shipping intuitive APIs, we want you on our team.
+- **Experience building and shipping production code in a team setting** with a passion for writing tested, performant, and high-quality code.
+- **Strong written and verbal communication skills**.
+  As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
```

Comment:
In this particular role, the majority of communication should be within the team.

```suggestion
  As part of the team, you'll interface both directly and indirectly with team members, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
```